docs/en/advanced_tutorials/visualization.md

```diff
@@ -1,3 +1,390 @@
+
+```python
+visualizer = Visualizer(vis_backends=[dict(type='LocalVisBackend')], save_dir='temp_dir')
+drawn_img = visualizer.draw_featmap(feat, image, channel_reduction=None, topk=5, arrangement=(2, 3))
+# temp_dir/vis_data/vis_image/feat_0.png will be generated
+visualizer.add_image('feat', drawn_img)
+```
+
+3. Save scalar data such as loss
+
+```python
+# temp_dir/vis_data/scalars.json will be generated
+# save loss
+visualizer.add_scalar('loss', 0.2, step=0)
+visualizer.add_scalar('loss', 0.1, step=1)
+# save acc
+visualizer.add_scalar('acc', 0.7, step=0)
+visualizer.add_scalar('acc', 0.8, step=1)
+```
+
+Multiple scalar data can also be saved at once.
+
+```python
+# New contents will be added to the temp_dir/vis_data/scalars.json
+visualizer.add_scalars({'loss': 0.3, 'acc': 0.8}, step=3)
+```
+
+4. Save configurations
+
+```python
+from mmengine import Config
+cfg=Config.fromfile('tests/data/config/py_config/config.py')
+# temp_dir/vis_data/config.py will be saved
+visualizer.add_config(cfg)
+```
+
+## Various Storage Backends
+
+Any `Visualizer` can be configured with any number of storage backends. `Visualizer` will traversal call the configured backends and save the results to each backend in a loop.
+
+```python
+visualizer = Visualizer(image=image, vis_backends=[dict(type='TensorboardVisBackend'),
+                                                   dict(type='LocalVisBackend')],
+                        save_dir='temp_dir')
+# temp_dir/vis_data/events.out.tfevents.xxx files will be generated
+visualizer.draw_bboxes(torch.tensor([[33, 120, 209, 220], [72, 13, 179, 147]]))
+visualizer.draw_texts("cat and dog", torch.tensor([10, 20]))
+visualizer.draw_circles(torch.tensor([40, 50]), torch.tensor([20]))
+
+visualizer.add_image('demo', visualizer.get_image())
+```
+
+Note: If there are multiple backends used at the same time, the `name` field must be specified. Otherwise, it is impossible to distinguish which backend it is.
+
+```python
+visualizer = Visualizer(image=image, vis_backends=[dict(type='TensorboardVisBackend', name='tb_1', save_dir='temp_dir_1'),
+                                                   dict(type='TensorboardVisBackend', name='tb_2', save_dir='temp_dir_2'),
+                                                   dict(type='LocalVisBackend', name='local')],
+                        save_dir='temp_dir')
+```
+
+## Visualize at Anywhere
+
+During the development, users may need to add visualization functions somewhere in their codes and save the results to different backends, which is very common for analysis and debugging. `Visualizer` in MMEngine can obtain the data from the same visualizers and then visualize them.
+
+Users only need to instantiate the visualization object through the `get_instance` during initialization. The visualization object at this stage is the only object that can be obtained globally. Then it can be obtained anywhere in the code through `Visualizer.get_current_instance()`.
```

Comment:
```suggestion
Users only need to instantiate the visualizer through `get_instance` during initialization. The visualizer obtained this way is unique and globally accessible. Then it can be accessed anywhere in the code through `Visualizer.get_current_instance()`.
```